packages/design-system/src/components/ButtonGroup/ButtonGroup.js

```diff
@@ -8,26 +8,26 @@ const baseClass = 'btn-group';
 
 class ButtonGroup extends React.Component {
   state = {
-    currentIndex: defaultIndex
+    currentindex: defaultIndex
```

Comment:
Why you removed the camelcase format?

Reply:
Yes I also opt for the original one that I can't find any reasoning for not making it in camel case as they are two separate words 🤔

Reply:
`const {... currentIndex, ...restProps } = this.props;` this fixed my issue, reverted curentIndex at b1005187a0d51f7e88f2f764c0701d21ca564965